src/dune/action.ml

```diff
@@ -302,6 +304,7 @@ let is_useful_to distribute memoize =
     | Dynamic_run _ -> true
     | System _ -> true
     | Bash _ -> true
+    | Format_dune_file _ -> memoize
```

Comment:
Wasn't sure about this one.

Reply:
That seems good

Reply:
Basically this means that:

- the action would not be sandboxed (no need since we don't run external commands)
- the result would go in the shared cache
- the result would go in the distributed shared cache

Reply:
Got it, thanks!